src/main/java/software/amazon/awssdk/crt/s3/S3MetaRequestResponseHandler.java

```diff
@@ -34,12 +34,12 @@ default int onResponseBody(ByteBuffer bodyBytesIn, long objectRangeStart, long o
 
     /**
      * Invoked when the entire meta request execution is complete.
-     *
-     * @param errorCode The CRT error code
-     * @param responseStatus statusCode of the HTTP response
-     * @param errorPayload body of the error response. Can be null if the request completed successfully
+     * @param context a wrapper object containing the following fields
+     * errorCode The CRT error code
+     * responseStatus statusCode of the HTTP response
+     * errorPayload body of the error response. Can be null if the request completed successfully
```

Comment:
remove these out dated documentation

Reply:
I just added this documentation, forgot to add a couple of points though.

Reply:
```
     * errorCode The CRT error code
     * responseStatus statusCode of the HTTP response
     * errorPayload body of the error response. Can be null if the request completed successfully
     * checksumAlgorithm, the algorith used to validate the Body, None if not validated
     * isChecksumValidated which is true if the response was validated.
```
Move those docs to the `S3FinishedResponseContext` instead. They are details about `S3FinishedResponseContext`.